tests/integration_test_reponse_templates.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from syncano.models import ResponseTemplate
+from tests.integration_test import InstanceMixin, IntegrationTest
+
+
+class ResponseTemplateApiTest(InstanceMixin, IntegrationTest):
+
+    @classmethod
+    def setUpClass(cls):
+        super(ResponseTemplateApiTest, cls).setUpClass()
+        cls.to_delete = cls.instance.templates.create(name='to_delete', content="<br/>",
+                                                      content_type='text/html', context={'one': 1})
+        cls.for_update = cls.instance.templates.create(name='to_update', content="<br/>",
+                                                       content_type='text/html', context={'one': 1})
+
+    def test_retrieve_api(self):
+        template = ResponseTemplate.please.get(name='to_update')
+        self.assertTrue(isinstance(template, ResponseTemplate))
+        self.assertTrue(template.name)
```

Comment:
I can check if name match here :) no problemo, want to avoid changing tests when ResponeTemplates change - but probably the name will be with us forever ;)